KubeArmor/build/kubearmor-test-docker.yaml

```diff
@@ -50,7 +50,7 @@ spec:
         imagePullPolicy: Never
         securityContext:
           privileged: true
-        command: ["/KubeArmor/kubearmor"]
+
```

Comment:
don't we need the command?

Reply:
The command is actually /KubeArmor/entrypoint.sh, which is already updated in the  KubeArmor/build/Dockerfile.kubearmor